cms/djangoapps/contentstore/views/item.py

```diff
@@ -128,6 +132,9 @@ def xblock_handler(request, usage_key_string):
               these fields:
                 :parent_locator: parent for new xblock, required for both duplicate and create new instance
                 :duplicate_source_locator: if present, use this as the source for creating a duplicate copy
+                :move_source_locator: if present, use this as the source item for moving
```

Comment:
I just have renamed it keeping the duplicate naming in mind. Like `duplicate_source_locator`, we have `move_source_locator`. If you want me to change, I can change it to `move_locator`.